src/MOI/MOI_wrapper.jl

```diff
@@ -138,6 +138,9 @@ mutable struct Optimizer <: MOI.AbstractOptimizer
     # parameter.
     silent::Bool
 
+    has_solution::Bool
```

Comment:
Do you really need this member? From what I can see, it is semantically equivalent to `variable_primal === nothing`, is always updated at the same time (except in `MOI.optimize!`), and never read.

Reply:
@Gbrunieau please remove `has_solution`. I just needed that for Cbc because the `ResultCount` check was slow.

Reply:
you're right, thanks, it's not needed, I updated the code